source/basic.tex

```diff
@@ -3056,7 +3056,11 @@
 A byte is at least large enough to contain any member of the basic
 \indextext{character set!basic execution}%
 execution character set\iref{lex.charset}
-and the eight-bit code units of the Unicode UTF-8 encoding form
+and the eight-bit code units of the Unicode%
+  \footnote{Unicode\textregistered\ is a registered trademark of Unicode, Inc.
+  This information is given for the convenience of users of this document and
+  does not constitute an endorsement by ISO or IEC of this product.}
+UTF-8 encoding form
```

Comment:
It seems a bit silly to talk about UTF-8 at all here, now that we guarantee 2's complement. "eight-bit" is all we need to say; anything else is irrelevant rationalization.

But let's go with what you have for C++20; we can reconsider this for C++23.